src/omero_zarr/masks.py

```diff
@@ -178,6 +190,8 @@ class MaskSaver:
     masks to zarr groups/arrays.
     """
 
+    OVERLAPS = ["error", "dtype_max"]
```

Comment:
As a very small aside, unless you want this to be externally extensible, a `tuple` property would be safer.

Reply:
fixed in [2a0c9ce](https://github.com/ome/omero-cli-zarr/pull/120/commits/2a0c9ce39c949cac65b2207bfe2a41dbfccc894a)